libdvdcss.mk

```diff
@@ -0,0 +1,50 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS        += libdvdcss
```

Comment:
I'm not changing this spacing for others anymore, please fix before merge. I'm writing a style guide soon literally because of this one space right here. I hope you're happy Nick.

Reply:
[`style(9)`](https://www.freebsd.org/cgi/man.cgi?query=style&apropos=0&sektion=9&manpath=FreeBSD+12.2-RELEASE+and+Ports&arch=default&format=html) for Procursus?